packages/tools/webpack-fluid-loader/README.md

```diff
@@ -75,9 +77,34 @@ or in an optional `config.json` file in the `baseDir` passed into `webpack-fluid
 
 ```
 
+## Azure Fluid Relay
+To use Azure Fluid Relay, [provision a Fluid Relay instance](https://learn.microsoft.com/en-us/azure/azure-fluid-relay/how-tos/provision-fluid-azure-portal) in the Azure Portal.
+To select the deployment region, consult [the availability table](https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=fluid-relay) for the Azure Fluid Relay product<sup id="a1">[1](#f1)</sup>.
```

Comment:
I _think_ you can remove the en-us from the URLs and folks will be redirected to a localized page (if one's available).

Reply:
Seems to work.  Nice suggestion!